course3/homework-linkedlist/my_linked_list.h

```diff
@@ -0,0 +1,119 @@
+#pragma once
+#include <cstdlib>
+#include <iostream>
+#include <memory>
+
+template<typename T>
+class MyLinkedList {
+	struct Node {
+		T mData;
+		std::unique_ptr<Node> mNext;
+		Node(const T& value) : mData(value) {}
+	};
+
+	public:
```

Comment:
There is too much indentation, and it's inconsistent (Node is indented once, iterator twice)